File: python/6.Sorting/Triangle.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(A):
    # Implement your solution here
        A.sort()
        for i in range(2, len(A)):
            P = A[i-2]
            Q = A[i-1]
            R = A[i]
            if P+Q>R and Q+R>P and R+P>Q:
                return 1
        return 0

File: csharp/6.Sorting/Triangle.cs
using System;
// you can also use other imports, for example:
// using System.Collections.Generic;

// you can write to stdout for debugging purposes, e.g.
// Console.WriteLine("this is a debug message");

class Solution {
    public int solution(int[] A) {
        Array.Sort(A);
        for (int i=2; i<A.Length; i++)
        {
            long P = A[i-2];
            long Q = A[i-1];
            long R = A[i];
            if(P+Q>R && Q+R>P && R+P>Q)
            {
                return 1;
            }
        }
        return 0;
    }
}

